src/Selenium2Driver.php

```diff
@@ -260,7 +213,8 @@ protected static function charToOptions($char, $modifier = null)
      */
     protected function executeJsOnXpath($xpath, $script, $sync = true)
```

Comment:
As it is a new major version, I suggest switching to private too (we don't want to make those supported extension points)

Reply:
done